src/components/Rating.jsx

```diff
@@ -177,28 +214,29 @@ export default class Rating extends Component {
   }
 
   /**
-   * On focus callback will enable focus styling on component when focused using tab.
+   * enable focus styling of component when tab is used to focus component
    */
   _onFocus() {
     if(!this.active) {
       this.setState({focused: true});
     }
-    this.forceUpdate();
     if (this.props.onFocus) {
       this.props.onFocus(event);
     }
   }
 
   /**
-   * When user clicks the component this callback will update component value to that selected by user.
+   * update component when clicked
    */
   _onClick(event) {
     if(!this.props.disabled) {
       this._updateComponent();
     }
+
     if (this.props.onClick) {
       this.props.onClick(event);
     }
+    React.findDOMNode(this.refs.wrapper).blur();
```

Comment:
Did you call blur to make sure focus is not applied on click?

This would break the default behaviour of all native & belle input elements. If you click on an element it's focused. For example with a native select you can open the dropdown with space. A checkbox can be toggled with space.